fredie7/sendIT
# parcel database setup

#### What does this PR do?
This PR ensures that a databse of parcels is setup for interaction.

#### Description of Tasks
- [ ] Ensures that a database is setup to perform all CRUD operations. 

#### How can I test it?
download postgres
run createdb sendit on the terminal
connect to database
run a check to ensure that it's connected
#### Background Context
ensure to have a .env file setup with DATABASE_URL
DATABASE_URL = postgres://xxxieflz:xBSu_8kU2N0eOjbSyEvNHqpP5_FvhqMc@drona.db.elephantsql.com:5432/xxxieflz
#### Screenshot
none
